prow/test/integration/test.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd -P )"
+
+if [[ -n "${CI:-}" ]]; then
+  # TODO(chaodaiG): remove this once kind is installed in test image
+  echo "Install KIND for prow"
+  curl -Lo /usr/bin/kind https://kind.sigs.k8s.io/dl/v0.9.0/kind-linux-amd64
+  chmod +x /usr/bin/kind
+
+  # TODO(chaodaiG): remove this once bazel is installed in test image
+  echo "Install bazel for prow"
```

Comment:
Same here, checking for thhe presence of bazel rather than for running in CI makes IMHO more sense

Reply:
I still don't think this works? If I have bazel installed, but don't have the correct bazel version, this will download the correct version, but then continue to use the version I originally had installed.
What is the need for requiring such a specific bazel version?

Reply:
This solves the problem of the test image has bazel installed but not at the version required, it felt to me that bazel is smart enough to figure out which version to use?